advection_solver.cc

```diff
@@ -985,6 +985,8 @@ namespace DGAdvection
     else
 #endif
       triangulation = std::make_shared<Triangulation<dim>>();
+    
+    dof_handler.reinit(*triangulation);
```

Comment:
The miniapp supports also deal.II 9.2 where this function is not present, so we need
```suggestion
#if DEAL_II_VERSION_GTE(9, 3, 0)
    dof_handler.reinit(*triangulation);
    dof_handler.distribute_dofs(fe);
#else
    dof_handler.initialize(*triangulation, fe);
#endif
(in the `setup_dofs` function).
```

Reply:
OK. I'll do that later. I am busy porting the application to simplex ;)